### PR TITLE
Add manual legend to MapCard

### LINKED ISSUE #218: MapChart - toggling between breakdown should have common legend
We should have an explicit legend passed into the map card if we're going to toggle the heat map based on a breakdown variable otherwise it's misleading

### PR DESCRIPTION
**Pull request by kkatzen**
Closes #218 
#370

### CODE REVIEW

File: frontend/src/data/query/MetricQuery.ts [hunk 1]
```diff
@@ -65,10 +65,21 @@ export class MetricQueryResponse {
...
+    if (fieldValues.length === 0) {
+      return undefined;
     }
+    return {
+      min: Math.min.apply(Math, fieldValues),
+      max: Math.max.apply(Math, fieldValues),
+    };
+  }
+
+  getUniqueFieldValues(fieldName: string): string[] {
```
Comment (aaronsn) [inline frontend/src/data/query/MetricQuery.ts:82]: Did you mean to drop the `if (this.isFieldMissing(fieldName)) {` check?

File: frontend/src/charts/ChoroplethMap.tsx [hunk 2]
```diff
@@ -14,6 +15,7 @@ const MISSING_DATASET = "MISSING_DATASET";
 const VALID_DATASET = "VALID_DATASET";
 const GEO_DATASET = "GEO_DATASET";
 const GEO_ID = "id";
+const COLOR_SCALE = "COLOR_SCALE";
```
Comment (aaronsn) [inline frontend/src/charts/ChoroplethMap.tsx:18]: Why did this change from "colorScale" to "COLOR_SCALE"? Does this work by referencing the below variable by the "name" property?

**Review by aaronsn — APPROVED** [review]

File: frontend/src/charts/ChoroplethMap.tsx (hunk 2 above)
Comment (kkatzen) [reply frontend/src/charts/ChoroplethMap.tsx:18]: Yeah I was trying to change the arbitrary string names to the variables to reduce human error.

File: frontend/src/data/query/MetricQuery.ts (hunk 1 above)
Comment (kkatzen) [reply frontend/src/data/query/MetricQuery.ts:82]: oops good catch

**Pull request merged**